cypress/local-only/tests/integration/trialSessionWorkingCopy/trial-session-working-copy-filter.cy.ts

```diff
@@ -0,0 +1,38 @@
+import { loginAsColvin } from "cypress/helpers/authentication/login-as-helpers"
+
+
+describe('trials session working copies filtering', () => {
+	it('logins as judge colvin', () => {
+		loginAsColvin();
+	});
+	it('should have all docket numbers set to statusUnassigned on initial load', () => {
+		cy.get('[data-testid="trial-session-link"]').click();
+		cy.get('[data-testid="trial-location-link-959c4338-0fac-42eb-b0eb-d53b8d0195cc"]').click();
+		cy.get('[data-testid="trial-session-working-copy-filter-statusUnassigned').find('span').invoke('text').then((text) => {
+			expect(text).to.equal(' (3)');
+		});
```

Comment:
It feels like we are testing the style when we check for the preceding space. I would just expect the text to contain `(3)`. Not sure it is a big deal though.